usaspending_api/search/v2/elasticsearch_helper.py

```diff
@@ -28,6 +28,17 @@ def preprocess(keyword):
     return keyword
 
 
+def es_sanitizer(keyword):
```

Comment:
I would rather see functions named as actions. "es_sanitizer" sounds like a class, not a function.